packages/twitch-pubsub-client/src/SingleUserPubSubClient.ts

```diff
@@ -146,6 +148,29 @@ export class SingleUserPubSubClient {
 		return this._addListener('chat_moderator_actions', callback, 'channel:moderate', extractUserId(channelId));
 	}
 
+	/**
+	 * Adds a listener to undocumented events to the client.
```

Comment:
I would prefer if the documentation for this stayed as close as possible to the non-single-user equivalent.